common/devdoc/interlocked_hl_requirements.md

```diff
@@ -63,22 +63,40 @@ MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForVal
 ```
 
 `InterlockedHL_WaitForValue` waits for the value at a given address to be equal to a target value.
+[Note: `address_to_check` refers to `address`, `value_to_wait` refers to `value`, and `timeout_ms` refers to `milliseconds`]
```

Comment:
Why are we making a change the existing functions without changing the functionality?  This change just seems to be adding variables unnecessarily.

Reply:
Fixed.